docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -273,6 +278,24 @@ Hadoop CLI with an Alluxio URI.
 $ hadoop fs -ls alluxio:///directory
 ```
 
+#### HA Client URI for Alluxio Client
+
+Users can use the Alluxio URI to connect to an Alluxio HA cluster, by fully specifying the HA cluster information.
+To specify the multi-master information within the Alluxio URI, 
+use `alluxio://master_hostname_1:19998,master_hostname_2:19998,master_hostname_3:19998`
+To specify the ZooKeeper information within the Alluxio URI, use `alluxio://zk@<ZOOKEEPER_ADDRESS>`.
+
+For example, for many applications (e.g., Hadoop, HBase, Hive and Flink), you can use a comma as the
+delimiter for multiple addresses in the URI, like `alluxio://master_hostname_1:19998,master_hostname_2:19998,master_hostname_3:19998` 
+and `alluxio://zk@zkHost1:2181,zkHost2:2181,zkHost3:2181/path`.
+
+For some other applications (e.g., Spark), you need to use semicolons as the delimiter for multiple
+addresses, like `alluxio://master_hostname_1:19998;master_hostname_2:19998;master_hostname_3:19998` 
+and `alluxio://zk@zkHost1:2181;zkHost2:2181;zkHost3:2181/path`. 
+
+If you use the URI to specify the HA cluster information, the URI will take precedence, and ignore the
+related Alluxio HA mode configuration.
```

Comment:
specifying HA information in the URI is itself a type of configuration

can we be more specific, and say it takes precedence over all other forms of configuration, e.g. site properties or environment variables?

Reply:
Changed to 
```
Users can fully specify the HA cluster information in the URI to connect to an Alluxio HA cluster.
Configuration derived from the HA authority takes precedence over all other forms of configuration, e.g. site properties or environment variables.
```